### PR TITLE
3639676: Adding the cables validation report to the NDT merger plugin

### PR DESCRIPTION
**Pull request by ananalaghbar**

### CODE REVIEW

**Comment by lennybe** [comment]
bot:retest

**Review by ananalaghbar — COMMENTED** [review]
bot:retest

**Comment by ananalaghbar** [comment]
bot:retest

File: plugins/UFM_NDT_Plugin/ufm_sim_web_service/ndt-ui/src/app/ndt/views/settings-view/validators/address.validators.ts [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+
+    return null;
+  }
+
+}
+
+export function isValidIpv4Address(message?:string):ValidatorFn {
+  return (control:AbstractControl):{[key:string]:any} | null => {
+
+    var ipRegEx = new RegExp("^" + IP_V4_REGEX + "$");
```
Comment (haithamwj) [inline plugins/UFM_NDT_Plugin/ufm_sim_web_service/ndt-ui/src/app/ndt/views/settings-view/validators/address.validators.ts:71]: it should be const or let

File: plugins/UFM_NDT_Plugin/ufm_sim_web_service/ndt-ui/src/app/ndt/views/settings-view/validators/address.validators.ts [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+
+/**
+ * @desc used to check if the value is a valid ipv6 socket address. socket address is [ipv6]:port
+ * @param message
+ * @returns {function(AbstractControl): ({}|null)}
+ */
+export function isValidPort(message?:string):ValidatorFn {
+  return (control:AbstractControl):{[key:string]:any} | null => {
+
+    var ipRegEx = new RegExp("^" + PORT_REGEX + "$");
```
Comment (haithamwj) [inline plugins/UFM_NDT_Plugin/ufm_sim_web_service/ndt-ui/src/app/ndt/views/settings-view/validators/address.validators.ts:96]: use const instead

**Review by haithamwj — APPROVED** [review]

**Pull request merged**